lib/tasks/svg.rake

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+namespace :svg do
+  desc "rerender SVG for molecules  \n
+     args[:all] default to false, if true, all molecules will be rerendered, \n
+     if false, only molecules with missing SVG will be rerendered"
+  task :molecule, %i[all from to] => [:environment] do |_t, args|
+    #   if true, render all molecules, otherwise only render those that have missing svg
+    # args.with_ketcher is a boolean: default to true, always check ketcher rendering service is available
```

Comment:
Some commented examples in the file would be ideal to properly show the list of arguments and how they can be used.

Some clarification on the existence of the `with_ketcher` boolean argument is needed. It is only in the commented line 9, and isn't used in the code at all to check if the ketcher service is available before continuing.

Reply:
fx and moved to https://github.com/ComPlat/chemotion_ELN/pull/1308